deployment/RS01SUM2_Deploy.csv

```diff
@@ -52,13 +52,12 @@ TN407,,,,RS01SUM2-MJ01B-00-FLOBNC101,10,1,2022-08-20T12:15:59,2023-08-27T04:30:0
 TN407,,,,RS01SUM2-MJ01B-00-FLOBNM101,10,1,2022-08-22T19:15:20,2023-08-28T04:30:00,RS01SUM2-MJ01B-0000,ATAPL-65244-00024,ATAPL-70113-00003,44.569847,-125.146626,,775,775,,,
 TN407,,,,RS01SUM2-MJ01B-00-OSMOIA101,9,1,2022-08-20T12:03:00,2023-08-28T06:20:00,RS01SUM2-MJ01B-0000,ATAPL-65244-00024,ATAPL-58333-00001,44.569903,-125.146702,,774,774,position listed is location of sensor intake,,
 TN407,,,,RS01SUM2-MJ01B-11-CTDPFA110,4,1,2022-08-21T03:29:00,2023-08-24T00:00:00,RS01SUM2-MJ01B-0000,ATAPL-65244-00024,PIRSN-CTDPFA-10002,44.569826,-125.146690,,775,775,,,
-TN422,,,,RS01SUM2-MJ01B-00-FLOBNC101,11,1,2023-08-27T04:23:00,2024-08-21T10:19:54,RS01SUM2-MJ01B-0000,ATAPL-65244-00028,ATAPL-58334-00001,44.569838,-125.146653,,775,775,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth,,
-TN422,,,,RS01SUM2-MJ01B-00-FLOBNM101,11,1,2023-08-28T03:12:00,2024-08-21T10:19:54,RS01SUM2-MJ01B-0000,ATAPL-65244-00028,ATAPL-70113-00002,44.569847,-125.146626,,775,775,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth,,
-TN422,,,,RS01SUM2-MJ01B-00-OSMOIA101,10,1,2023-08-28T03:50:00,2024-08-21T10:19:54,RS01SUM2-MJ01B-0000,ATAPL-65244-00028,ATAPL-58333-00003,44.569903,-125.146702,,774,774,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth,,
-TN422,,,,RS01SUM2-MJ01B-05-CAMDSB103,8,1,2023-08-27T23:09:00,2024-08-20T17:04:44,RS01SUM2-MJ01B-0000,ATAPL-65244-00028,ATAPL-78437-00007,44.569998,-125.146713,,775,775,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth,,
-TN422,,,,RS01SUM2-MJ01B-12-ADCPSK101,3,1,2023-08-27T18:13:00,,RS01SUM2-MJ01B-0000,ATAPL-65244-00028,ATAPL-58419-00002,44.569849,-125.146783,,775,775,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth,,
+TN422,,,,RS01SUM2-MJ01B-00-FLOBNC101,11,1,2023-08-27T04:23:00,2024-08-21T10:19:54,RS01SUM2-MJ01B-0000,ATAPL-65244-00028,ATAPL-58334-00001,44.569979,-125.14668,,774,774,,, 
+TN422,,,,RS01SUM2-MJ01B-00-FLOBNM101,11,1,2023-08-28T03:12:00,2024-08-21T10:19:54,RS01SUM2-MJ01B-0000,ATAPL-65244-00028,ATAPL-70113-00002,44.570022,-125.146728,,775,775,,,
+TN422,,,,RS01SUM2-MJ01B-00-OSMOIA101,10,1,2023-08-28T03:50:00,2024-08-21T10:19:54,RS01SUM2-MJ01B-0000,ATAPL-65244-00028,ATAPL-58333-00003,44.56994,-125.146624,,774,774,,,
```

Comment:
I think this should be the intake position (with a comment like the older ones to indicate that): 44.569935	-125.146629